sunpy/map/mapbase.py

```diff
@@ -1726,7 +1747,16 @@ def plot(self, annotate=True, axes=None, title=True,
                               SunpyUserWarning)
 
         # Normal plot
-        imshow_args = copy.deepcopy(self.plot_settings)
+        imshow_args = {
+                        'cmap': self.cmap,
+                        'norm': self.norm,
+                        'interpolation': 'nearest',
+                        'origin': 'lower',
+                      }
+
+        if hasattr(self, 'plot_settings') and self.plot_settings is not None:
```

Comment:
```suggestion
        if self.plot_settings:
```

Given my above suggestion, we don't need to check for the existence of `self.plot_settings`.

Reply:
Is there a reason to check explicitly if it is not `None`?